frontend/src/metabase/containers/CollectionItemsLoader.jsx

```diff
@@ -13,7 +13,14 @@ const CollectionItemsLoader = ({ collectionId, children, ...props }: Props) => (
     {({ object }) => (
       <Search.ListLoader
         {...props}
-        query={{ collection: collectionId }}
+        query={{
+          collection: collectionId,
+          pinned_state: "is_pinned",
+          sort_column: "name",
+          sort_direction: "asc",
+          models: "dashboard",
+          limit: 200,
```

Comment:
@flamber @mazameli can you please share how pinned dashboards we can show on the home page under "Start here"?

<img width="1439" alt="Screen Shot 2021-07-07 at 16 31 14" src="https://user-images.githubusercontent.com/14301985/124767925-e6eb1880-df40-11eb-8cce-6467cfff4766.png">

Reply:
The same question about the number of folders under "Our analytics"

Reply:
Worth to be a constant value in the top of a file I guess

Reply:
Huh, good question. I'll throw out some opinions, but I defer to @flamber here; he knows better than I do what our real customers' upper bounds might be.

But, my answer for both is, "display as many as we can before we expect we'll see performance degredation." 

It looks like you set it to 200 here for dashboards, which seems fine enough to me — I find it hard to picture a scenario where an instance will have 200 _pinned_ dashboards all in the root folder.

I can imagine a super large instance theoretically having hundreds of top-level folders, so I might bump the cap up to like 500 just to give ourselves ample breathing room, provided that that doesn't cause performance problems.

Reply:
@mazameli I bumped it to 500 for now, thanks. Let's hear @flamber.
I assume it would not be convenient enough to have that many folders on the home page, however, this problem should be tackled in a different way

Reply:
@mazameli @flamber @rlotun it seems the question about the limits of pinned dashboards and collections on the home page is not a blocker and this PR already fixes an important issue for the `0.40` release, so I'll merge it now. We will easily change limits later if we need it